### DISCUSSION TITLE
MidFlow

### DISCUSSION THREAD

**Discussion by Kittusirkh** [huggingface/diffusers#9544] [General]
https://huggingface.co/enhanceaiteam/MidFlow 👋🥳 can you add this in diffusers pipeline

**Comment by asomoza** [comment]
Hi, I tried to test it with ComfyUI and didn't work. Do you have more information about this model? It's a different arch? What should be done to make it work with diffusers?

**Reply by Kittusirkh** [reply]
The layers are exactly the same as Flux.

https://huggingface.co/enhanceaiteam/MidFlow/tree/main/high

For the text encoder - [text_encoder](https://huggingface.co/enhanceaiteam/MidFlow/resolve/main/clip/clip_l.safetensors)

For the second text encoder - [text_encoder_2](https://huggingface.co/enhanceaiteam/MidFlow/resolve/main/text_encoder/t5xxl.safetensors)

For the VAE - [vae](https://huggingface.co/enhanceaiteam/MidFlow/resolve/main/vae/vae.safetensors)

**Reply by asomoza** [reply]
Thanks, if its the same as Flux, can't it just be used with the Flux Pipeline?

Also just a reminder, if you used Flux.dev as a base, you can't change the license to apache 2.0
